docs/self-managed/console-deployment/telemetry.md

```diff
@@ -0,0 +1,115 @@
+---
+id: telemetry
+title: "Console (Self-Managed) telemetry"
+sidebar_label: "Telemetry"
+description: "Console (Self-Managed) telemetry introduction"
+---
+
+At Camunda, we strive to offer an excellent user experience at a high and stable level. On a strictly opt-in basis, we are looking to collect environment and usage data to improve your user experience further. These insights help us understand typical environment setups and product usage patterns and will be used to make informed product improvement decisions that benefit you.
+
+## Design
+
+Camunda 8 Console Self-Managed incorporates a telemetry mechanism designed to periodically report usage data to Camunda. This telemetry operates seamlessly in the background and is activated only when telemetry is explicitly enabled. The data collection mechanism is structured to function without interrupting or degrading the performance of the implemented processes. When enabled, the collected data is sent once every 24 hours via HTTPS.
+
+Once telemetry is enabled, a new telemetry page will display collected information. This page can be used to view component metrics as a simple monitoring interface. Console will collect a subset of metrics, which you can download and inspect anytime.
```

Comment:
> Once telemetry is enabled, a new telemetry page will display collected information 

Where is this page located? Is this a UI reference?

Reply:
YEs, Console SM has new page called Telemetry. Should I explain is in more details?

Reply:
Yes please, and/or add a screenshot 👍

Reply:
Done